ipie/utils/from_trexio.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 The ipie Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors: Anthony Scemama <scemama@irsamc.ups-lse.fr>
+#
+
+"""Generate AFQMC data from a TREXIO file."""
+try:
+  import trexio
+except:
+  print("TREXIO is not installed. Try pip install trexio.")
+  raise
+
+import time
+from dataclasses import dataclass
```

Comment:
I know we're not currently ones to talk, but could you remove the unused imports here?

Reply:
and below too

Reply:
Done